.travis.yml

```diff
@@ -22,17 +22,33 @@ matrix:
   fast_finish: true
   include:
     -
-      bundler_args: 
+      bundler_args:
       dist: trusty
-      env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_set=docker/centos-7
+      env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_setfile=centos7-64{hypervisor=docker}
```

Comment:
Can we get this change reverted back to BEAKER_set=docker/centos-7 and BEAKER_set=docker/ubuntu-14.04 please? (Work is ongoing to implement this functionality.)

Reply:
done